src/bookshelf/src/bksw_conf.erl

```diff
@@ -129,6 +131,22 @@ port() ->
             {port, Port}
     end.
 
+rsync_uri() ->
+    case application:get_env(bookshelf, rsync_uri) of
+        undefined ->
+            undefined;
```

Comment:
To be honest, I was mostly just following the structure of the other methods here, with the only difference being that in this case that my "default" value is also undefined.